fdbclient/Schemas.cpp

```diff
@@ -755,6 +755,7 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
          "auto_logs":3,
          "commit_proxies":5,
          "grv_proxies":1,
+         "proxies":6,
```

Comment:
You'll need an analogous change in https://github.com/apple/foundationdb/blob/master/documentation/sphinx/source/mr-status-json-schemas.rst.inc

Reply:
Thanks! done